src/integrationTest/java/uk/gov/hmcts/reform/juddata/config/CamelConfig.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.hmcts.reform.juddata.config;
+
+import javax.sql.DataSource;
+import org.apache.camel.CamelContext;
+import org.apache.camel.spring.SpringCamelContext;
+import org.apache.camel.spring.spi.SpringTransactionPolicy;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.testcontainers.containers.PostgreSQLContainer;
+import uk.gov.hmcts.reform.juddata.camel.aggregate.ListAggregationStrategy;
+import uk.gov.hmcts.reform.juddata.camel.beans.JudicialOfficeAppointment;
+import uk.gov.hmcts.reform.juddata.camel.beans.JudicialUserProfile;
+import uk.gov.hmcts.reform.juddata.camel.mapper.JudicialOfficeAppointmentRowMapper;
+import uk.gov.hmcts.reform.juddata.camel.mapper.JudicialUserProfileRowMapper;
+import uk.gov.hmcts.reform.juddata.camel.processor.ExceptionProcessor;
+import uk.gov.hmcts.reform.juddata.camel.processor.FileReadProcessor;
+import uk.gov.hmcts.reform.juddata.camel.processor.JudicialOfficeAppointmentProcessor;
+import uk.gov.hmcts.reform.juddata.camel.processor.JudicialUserProfileProcessor;
+import uk.gov.hmcts.reform.juddata.camel.route.ParentOrchestrationRoute;
+
+@Configuration
+public class CamelConfig {
+
+    @Autowired
+    ApplicationContext applicationContext;
+
+    @Bean
+    ParentOrchestrationRoute parentRoute() {
+        return new ParentOrchestrationRoute();
+    }
+
+    @Bean
+    JudicialUserProfileProcessor judicialUserProfileProcessor() {
+        return new JudicialUserProfileProcessor();
+    }
+
+    @Bean
+    JudicialUserProfileRowMapper judicialUserProfileRowMapper() {
+        return new JudicialUserProfileRowMapper();
+    }
+
+    @Bean
+    JudicialUserProfile judicialUserProfile() {
+        return new JudicialUserProfile();
+    }
+
+    @Bean
+    FileReadProcessor fileReadProcessor() {
+        return new FileReadProcessor();
+    }
+
+    @Bean
+    ListAggregationStrategy listAggregationStrategy()  {
+        return new ListAggregationStrategy();
+    }
+
+
+    private static final PostgreSQLContainer testPostgres = new PostgreSQLContainer("postgres");
```

Comment:
can you set the db name on the container?

Reply:
it creates db name hence no need

Reply:
no.. i would suggest to define the db name so that its created with same name everytime and not a random name.. 
its optional and not mandatory to do so